lore/pipelines/holdout.py

```diff
@@ -90,6 +90,12 @@ def encoders(self):
         if self._encoders is None:
             with timer('fit encoders'):
                 self._encoders = self.get_encoders()
+
+                # Ensure we have an iterable for all single encoder cases
+                if self._encoders.__class__ is not tuple:
```

Comment:
Can we make this a check for other iterables, like array? i.e.
```
import collections
if not isinstance(self._encoders, collections.Iterable):
```

Reply:
👍